### DISCUSSION TITLE
A force closed channel should be explained to the user

### DISCUSSION THREAD

**Discussion by holzeis** [get10101/10101#1078] [Ideas]
We could show the channel state to the user using the newly introduced status screen. We could use that as well to tell the user if the channel got force closed, and e.g. indicate when the on-chain funds get back.

Maybe it would also make sense to have a dedicated screen depicting channel state, but we are missing some "real estate" to add such screens.

**Comment by bonomat** [comment]
Channel state is an interesting problem because there could be multiple channels at any point of time, i.e. one channel could be in closing state while another one is just being opened. 

What do you think of showing this in the tx details? E.g. the tx details tell that this tx belongs to a channel opening or channel closing.

**Comment by holzeis** [comment]
I think we can also show it in the tx details. We may have to think about how this information will be shown to the user.

The need for this ticket originated for me because only seeing 0 balances without any transactions is a bit scary.

In other words I think it's necessary to show the user what's going 🙂.

**Comment by luckysori** [comment]
I think we could use both:

1. Dedicated screen to display current (and former) channel state.
2. A transaction history that shows that, for example, a split transaction has just been published on-chain because the channel was force-closed.

The second point is complex because we would need to introduce new sources for transactions: transactions published by LDK and `rust-dlc`. And these transactions have a different meaning since they don't pay directly to us or spend an output of ours. It's really not obvious how to display this sensibly.

Thinking about it I am more in favour of the first option at this stage. We could:

- Display the status: open, closing, closed.
- Dump a series of events that we know have happened e.g.: channel force-closed; failed to publish our LN commitment TX; counterparty LN commitment TX confirmed; spotted spendable output; spent spendable output in TX XYZ.

**Comment by Restioson** [comment]
> I think we can also show it in the tx details. We may have to think about how this information will be shown to the user.

> A transaction history that shows that, for example, a split transaction has just been published on-chain because the channel was force-closed.

Related: #872

**Comment by klochowicz** [comment]
> Display the status: open, closing, closed.

This could be added to the `Status` screen on the right in a way that's not too hard.
